### PR TITLE
Improve de-DE (German) Translation

### PR DESCRIPTION
**Pull request by gbakeman**
Update the German translation. Primary pull request is based on Lineflyer's https://github.com/gawindx/WinNUT-Client/pull/108.

Related PRs: https://github.com/gawindx/WinNUT-Client/pull/80, https://github.com/gawindx/WinNUT-Client/pull/131

### CODE REVIEW

File: Translation/de-DE/de-DE.csv [hunk 1]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Lineflyer,Minimum Output Voltage. Accepted value: Numeric value from 0 to 999.,Minimale Ausgangsspannung. Wertebereich: Numerischer Wert von 0 bis 999.
+Valid,Gawindx,Miscellanous,Verschiedenes
+Valid,Gawindx,Monthly,Monatlich
+Valid,Gawindx,Name :,Name :
+Valid,Gawindx,No Update Available,Kein Update verfügbar
+Valid,Gawindx,Not Connected,Nicht verbunden
+Valid,Lineflyer,Notice,Hinweis
+Valid,Lineflyer,NUT host,NUT-Host
+Valid,Lineflyer,NUT Port,NUT-Port
+Valid,Lineflyer,Nut server address. Accepted value: IPV4 / IPV6 / FQDN address.,Nut-Serveradresse. Wertebereich: IPV4 / IPV6 / FQDN-Adresse.
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:95]:
```suggestion
Valid,Lineflyer,Nut server address. Accepted value: IPV4 / IPV6 / FQDN address.,NUT-Serveradresse. Wertebereich: IPV4 / IPV6 / FQDN-Adresse.
```

File: Translation/de-DE/de-DE.csv [hunk 2]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Indicates whether a shutdown should be performed when the NUT server signals a forced shutdown (FSD = Force ShutDown),Gibt an ob ein Herunterfahren durchgeführt werden soll wenn der NUT-Server ein erzwungenes Herunterfahren signalisiert (FSD = Force ShutDown).
+Valid,Gawindx,Input Frequency,Eingangsfrequenz
+Valid,Gawindx,Input Voltage,Eingangsspannung
+Valid,Gawindx,Item Properties,Eigenschaften
+Valid,Lineflyer,List UPS Variables,Liste der USV-Variablen
+Valid,Lineflyer,Log file log level,Detailstufe für die Logdatei
+Valid,Lineflyer,Logging level,Detailstufe
+Valid,Lineflyer,Logging level.,Detailstufe.
+Valid,Lineflyer,Login,Anmeldung
+Valid,Lineflyer,Login credentials to the Nut server (leave blank if not required),Anmeldeinformationen des Nut-Server (Leer lassen wenn nicht erforderlich)
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:67]:
```suggestion
Valid,Lineflyer,Login credentials to the Nut server (leave blank if not required),Anmeldeinformationen des NUT-Server (Leer lassen wenn nicht erforderlich)
```

File: Translation/de-DE/de-DE.csv [hunk 3]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Lineflyer,Create LogFile,Logdatei erstellen
+Valid,Gawindx,Daily,Täglich
+Valid,Gawindx,Debug,Debuggen
+Valid,Lineflyer,Delay Between Verification,Verzögerung zwischen Überprüfungen
+Valid,Lineflyer,Delay to Shutdown (sec),Verzögerung vor dem Herunterfahren (Sek.)
+Valid,Lineflyer,Delete log file.,Logdatei löschen.
+Valid,Gawindx,Description :,Beschreibung :
+Valid,Gawindx,Devellopement,Entwicklung
+Valid,Gawindx,Disconnect,Trennen
+Valid,Lineflyer,Disconnected from Nut Host,Vom Nut-Host getrennt
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:40]:
```suggestion
Valid,Lineflyer,Disconnected from Nut Host,Vom NUT-Host getrennt
```

File: Translation/de-DE/de-DE.csv [hunk 4]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Lineflyer,NUT host,NUT-Host
+Valid,Lineflyer,NUT Port,NUT-Port
+Valid,Lineflyer,Nut server address. Accepted value: IPV4 / IPV6 / FQDN address.,Nut-Serveradresse. Wertebereich: IPV4 / IPV6 / FQDN-Adresse.
+Valid,Gawindx,NUT server connection identifier,Verbindungskennung des NUT-Servers
+Valid,Gawindx,NUT server login password,Anmeldekennwort des NUT-Servers
+Valid,Lineflyer,Nut Server Port Number (default = 3493). Accepted value: Numeric value from 1 to 65535.,Nut-Server Portnummer (Standard = 3493). Wertebereich: Numerischer Wert von 1 bis 65535.
+Valid,Gawindx,Ok,OK
+Valid,Gawindx,Old ups.ini imported,Alte ups.ini importiert
+Valid,Lineflyer,Old ups.ini imported Ini File Moved to {0}.old,Alte ups.ini importiert INI-Datei nach {0}.old verschoben
+To validate,Lineflyer,On Battery ({0}%),Auf Batterie ({0}%)
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:102]:
```suggestion
To validate,Lineflyer,On Battery ({0}%),Im Batteriebetrieb ({0}%)
```

File: Translation/de-DE/de-DE.csv [hunk 5]
```diff
@@ -1,161 +1,161 @@
...
+To validate,Lineflyer,On Line,Auf Netzbetrieb
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:103]:
```suggestion
To validate,Lineflyer,On Line,Im Netzbetrieb
```

File: Translation/de-DE/de-DE.csv [hunk 6]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Lineflyer,Shutdown if runtime lower than (sec),Herunterfahren wenn Laufzeit niedriger als (Sek.)
+Valid,Lineflyer,Shutdown on NUT's FSD Signal,Herunterfahren bei FSD-Signal des NUT
+Valid,Gawindx,Shutdown Options,Optionen zum Herunterfahren
+Valid,Lineflyer,Sleep,Energie sparen
+Valid,Gawindx,Stable,Stabil
+Valid,Lineflyer,Stable Or Dev Branch,Stabiler- oder Entwicklungszweig
+Valid,Gawindx,Start Minimized,Minimiert starten
+Valid,Lineflyer,Start with Windows,Mit Windows starten
+Valid,Gawindx,Starts the shutdown procedure immediately or timed.,Startet das Herunterfahren sofort oder zeitgesteuert.
+Valid,Lineflyer,Starts WinNut Minimized.,Startet WinNut minimiert.
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:132]:
```suggestion
Valid,Lineflyer,Starts WinNut Minimized.,Startet WinNUT minimiert.
```

File: Translation/de-DE/de-DE.csv [hunk 7]
```diff
@@ -1,161 +1,161 @@
...
+To validate,Lineflyer,Stop procedure delay if delayed. Accepted value: Numeric value from 0 to 3600.,Prozedurverzögerung bei Verzögerung stoppen. Wertebereich: Numerischer Wert von 0 bis 3600.
+Valid,Lineflyer,Time between each update of UPS data (default = 5).Accepted value: Numerical value from 1 to 60 seconds.,Zeit zwischen jeder Aktualisierung der USV-Daten (Standard = 5).Wertebereich: Numerischer Wert von 1 bis 60 Sekunden.
+Valid,Lineflyer,Try {0} of {1},Versuch {0} von {1}
+Valid,Lineflyer,Try Reconnect {0} / {1},Versuche Wiederverbindung {0} / {1}
+Valid,Gawindx,Type of Stop,Art des Stopps
+Valid,Gawindx,Type of stop at the end of the stop procedure.,Art des Stopps am Ende des Stoppvorgangs.
+Valid,Gawindx,Unknown UPS Name,Unbekannter USV-Name
+Valid,Gawindx,Update,Aktualisieren
+Valid,Gawindx,Update Available : Version {0},Update verfügbar : Version {0}
+Valid,Lineflyer,Update WinNUT-Client Now? Ok to Close WinNut and Install New Version Cancel to Save Msi and Install Later,WinNUT-Client jetzt aktualisieren? OK um WinNut zu schließen und neue Version zu installieren Abbrechen um MSI-Datei zu speichern und später zu installieren
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:144]:
```suggestion
Valid,Lineflyer,Update WinNUT-Client Now? Ok to Close WinNut and Install New Version Cancel to Save Msi and Install Later,WinNUT-Client jetzt aktualisieren? OK um WinNUT zu schließen und neue Version zu installieren Abbrechen um MSI-Datei zu speichern und später zu installieren
```

File: Translation/de-DE/de-DE.csv [hunk 8]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,UPS Battery Low,USV-Batterie schwach
+Valid,Lineflyer,UPS Load,USV-Last
+Valid,Lineflyer,UPS Name,USV-Name
+Valid,Lineflyer,UPS name to monitor. Accepted Value: Name of the UPS configured in the Nut server.,Name der zu überwachenden USV. Wertebereich: Name der auf dem Nut-Server konfigurierten USV.
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:148]:
```suggestion
Valid,Lineflyer,UPS name to monitor. Accepted Value: Name of the UPS configured in the Nut server.,Name der zu überwachenden USV. Wertebereich: Name der auf dem NUT-Server konfigurierten USV.
```

File: Translation/de-DE/de-DE.csv [hunk 9]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Lineflyer,UPS On Line,USV im Netzbetrieb
+Valid,Gawindx,UPS Overload,USV-Überlastung
+Valid,Lineflyer,UPS Variable,USV-Variablen
+Valid,Lineflyer,Use the Stable or Devellop branch during verification.,Bei der Versionsüberprüfung den stabilen oder Entwicklungszweig verwenden.
+Valid,Gawindx,Value :,Wert :
+Valid,Lineflyer,Verify Update,Auf Update prüfen
+Valid,Lineflyer,Verify Update At Start,Beim Start auf Updates prüfen
+Valid,Lineflyer,Version {0} Available,Version {0} verfügbar
+Valid,Gawindx,Weekly,Wöchentlich
+Valid,Gawindx,WinNut Preferences Changed,WinNut-Einstellungen geändert
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:159]:
```suggestion
Valid,Gawindx,WinNut Preferences Changed,WinNUT-Einstellungen geändert
```

File: Translation/de-DE/de-DE.csv [hunk 10]
```diff
@@ -1,161 +1,161 @@
...
+To validate,Lineflyer,Cancellation of the Extinguishing Process,Abbruch des Löschprozesses
+Valid,Gawindx,Check for updates.,Auf Updates prüfen.
+Valid,Gawindx,Checks for updates at startup.,Sucht beim Start nach Updates.
+Valid,Lineflyer,Clear,Leeren
+Valid,Gawindx,Close,Schließen
+Valid,Lineflyer,Close to Systray,Auf Systray minimieren
+To validate,Lineflyer,Conditions met for launching the Extinguishing Process,Voraussetzungen für den Start des Löschprozesses liegen vor
+Valid,Lineflyer,Connected,Verbunden
+Valid,Gawindx,Connection,Verbindung
+Valid,Gawindx,Connection to Nut Host {0}:{1} Established,Verbindung zum Nut Host {0}:{1} hergestellt
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:27]:
```suggestion
Valid,Gawindx,Connection to Nut Host {0}:{1} Established,Verbindung zum NUT-Host {0}:{1} hergestellt
```

File: Translation/de-DE/de-DE.csv [hunk 11]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Connection to Nut Host {0}:{1} Failed: {2},Verbindung zum Nut-Host {0}:{1} fehlgeschlagen: {2}
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:28]:
```suggestion
Valid,Gawindx,Connection to Nut Host {0}:{1} Failed: {2},Verbindung zum NUT-Host {0}:{1} fehlgeschlagen: {2}
```

File: Translation/de-DE/de-DE.csv [hunk 12]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Name :,Name :
+Valid,Gawindx,No Update Available,Kein Update verfügbar
+Valid,Gawindx,Not Connected,Nicht verbunden
+Valid,Lineflyer,Notice,Hinweis
+Valid,Lineflyer,NUT host,NUT-Host
+Valid,Lineflyer,NUT Port,NUT-Port
+Valid,Lineflyer,Nut server address. Accepted value: IPV4 / IPV6 / FQDN address.,Nut-Serveradresse. Wertebereich: IPV4 / IPV6 / FQDN-Adresse.
+Valid,Gawindx,NUT server connection identifier,Verbindungskennung des NUT-Servers
+Valid,Gawindx,NUT server login password,Anmeldekennwort des NUT-Servers
+Valid,Lineflyer,Nut Server Port Number (default = 3493). Accepted value: Numeric value from 1 to 65535.,Nut-Server Portnummer (Standard = 3493). Wertebereich: Numerischer Wert von 1 bis 65535.
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:98]:
```suggestion
Valid,Lineflyer,Nut Server Port Number (default = 3493). Accepted value: Numeric value from 1 to 65535.,NUT-Server Portnummer (Standard = 3493). Wertebereich: Numerischer Wert von 1 bis 65535.
```

File: Translation/de-DE/de-DE.csv [hunk 13]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Type of stop at the end of the stop procedure.,Art des Stopps am Ende des Stoppvorgangs.
+Valid,Gawindx,Unknown UPS Name,Unbekannter USV-Name
+Valid,Gawindx,Update,Aktualisieren
+Valid,Gawindx,Update Available : Version {0},Update verfügbar : Version {0}
+Valid,Lineflyer,Update WinNUT-Client Now? Ok to Close WinNut and Install New Version Cancel to Save Msi and Install Later,WinNUT-Client jetzt aktualisieren? OK um WinNut zu schließen und neue Version zu installieren Abbrechen um MSI-Datei zu speichern und später zu installieren
+Valid,Gawindx,UPS Battery Low,USV-Batterie schwach
+Valid,Lineflyer,UPS Load,USV-Last
+Valid,Lineflyer,UPS Name,USV-Name
+Valid,Lineflyer,UPS name to monitor. Accepted Value: Name of the UPS configured in the Nut server.,Name der zu überwachenden USV. Wertebereich: Name der auf dem Nut-Server konfigurierten USV.
+Valid,Gawindx,UPS On Battery,USV im Akkubetrieb
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:149]:
```suggestion
Valid,Gawindx,UPS On Battery,USV im Batteriebetrieb
```

File: Translation/de-DE/de-DE.csv [hunk 14]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Battery_Charge : {0} Remaining Time : {1},Batterieladung: {0} Verbleibende Zeit: {1}
+Valid,Gawindx,Battery_Charge : {WinNUT.UPS_BattCh} Remaining Time : {WinNUT.Lbl_VRTime.Text},Batterieladung: {WinNUT.UPS_BattCh} Verbleibende Zeit: {WinNUT.Lbl_VRTime.Text}
+Valid,Gawindx,Calibration,Kalibrierung
+Valid,Lineflyer,Cancel,Abbrechen
+To validate,Lineflyer,Cancellation of the Extinguishing Process,Abbruch des Löschprozesses
+Valid,Gawindx,Check for updates.,Auf Updates prüfen.
+Valid,Gawindx,Checks for updates at startup.,Sucht beim Start nach Updates.
+Valid,Lineflyer,Clear,Leeren
+Valid,Gawindx,Close,Schließen
+Valid,Lineflyer,Close to Systray,Auf Systray minimieren
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:23]:
```suggestion
Valid,Lineflyer,Close to Systray,In Systray minimieren
```

File: Translation/de-DE/de-DE.csv [hunk 15]
```diff
@@ -1,161 +1,161 @@
...
+Valid,Gawindx,Manufacturer :,Hersteller:
+Valid,Gawindx,Max,Max
+Valid,Lineflyer,Max Retry reached. Wait for manual Reconnection,Maximale Anzahl Verbindungsversuche erreicht. Auf manuelle Wiederverbindung warten.
+Valid,Lineflyer,Maximum Battery Voltage. Accepted value: Numeric value from 0 to 100.,Maximale Batteriespannung. Wertebereich: Numerischer Wert von 0 bis 100.
+Valid,Lineflyer,Maximum input frequency. Accepted value: Numeric value from 0 to 100.,Maximale Eingangsfrequenz. Wertebereich: Numerischer Wert von 0 bis 100.
+Valid,Lineflyer,Maximum input voltage. Accepted value: Numeric value from 0 to 999.,Maximale Eingangsspannung. Wertebereich: Numerischer Wert von 0 bis 999.
+Valid,Lineflyer,Maximum Load Level of the UPS. Accepted value: Numeric value from 0 to 100.,Maximaler Last der USV. Wertebereich: Numerischer Wert von 0 bis 100.
+Valid,Lineflyer,Maximum Output Voltage. Accepted value: Numeric value from 0 to 999.,Maximale Ausgangsspannung. Wertebereich: Numerischer Wert von 0 bis 999.
+Valid,Gawindx,Min,Min
+Valid,Gawindx,Minimize to Systray,Auf Systray minimieren
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:81]:
```suggestion
Valid,Gawindx,Minimize to Systray,In Systray minimieren
```

File: Translation/de-DE/de-DE.csv [hunk 16]
```diff
@@ -1,161 +1,161 @@
...
+To validate,Lineflyer,On Battery ({0}%),Auf Batterie ({0}%)
+To validate,Lineflyer,On Line,Auf Netzbetrieb
+Valid,Lineflyer,Open the log file.,Logdatei öffnen.
+Valid,Gawindx,Options,Optionen
+Valid,Gawindx,Output Voltage,Ausgangsspannung
+Valid,Gawindx,Password,Passwort
+Valid,Gawindx,Polling Interval,Abrufintervall
+Valid,Gawindx,Reconnect,Erneut verbinden
+Valid,Gawindx,Reconnection In Progress,Wiederverbindung läuft
+Valid,Lineflyer,Reduced to the taskbar or Systray.,Auf Taskleiste oder Systray minimieren.
```
Comment (MarkusMDK) [inline Translation/de-DE/de-DE.csv:111]:
```suggestion
Valid,Lineflyer,Reduced to the taskbar or Systray.,In Taskleiste oder Systray minimieren.
```

**Review by MarkusMDK — CHANGES_REQUESTED** [review]
Very good translation. I only have a few small suggestions for changes.

**Comment by gbakeman** [comment]
Great work, thank you @MarkusMDK.

**Pull request merged**